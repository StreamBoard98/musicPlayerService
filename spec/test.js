import React from 'react';
import Enzyme, { shallow, mount, render } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { createSerializer } from 'enzyme-to-json';
import sinon from 'sinon';

import MusicPlayer from '../client/src/components/musicPlayer.jsx';
import AlbumModal from '../client/src/components/albumModal.jsx';

expect.addSnapshotSerializer(createSerializer({ mode: 'deep' }));

Enzyme.configure({ adapter: new Adapter() });

it('renders correctly', () => {
  const wrapper = shallow(
    <MusicPlayer />
  );

  expect(wrapper).toMatchSnapshot();
});



// import puppeteer from 'puppeteer';

// const pageUrl = 'http://localhost:6969/';

// let page;
// let browser;
// const width = 1280;
// const height = 720;

// beforeAll(async () => {
//   browser = await puppeteer.launch({
//     headless: false,
//     slowMo: 80,
//     args: [`--window-size=${width},${height}`]
//   });
//   page = await browser.newPage();
//   await page.setViewport({ width, height });
// });

// afterAll(() => {
//   browser.close();
// });
