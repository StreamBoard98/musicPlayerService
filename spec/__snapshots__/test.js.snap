// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<styled.div>
  <styled.div>
    <styled.div
      onClick={[Function]}
    >
      <styled.img
        src="https://s3-us-west-1.amazonaws.com/streamboard98/icons/play.png"
      />
    </styled.div>
    <styled.div>
      <styled.div>
        <styled.span
          onClick={[Function]}
        />
      </styled.div>
      <styled.div>
        <styled.span />
      </styled.div>
      <styled.div>
        <styled.img
          src="https://s3-us-west-1.amazonaws.com/streamboard98/albumCovers/undefined"
        />
        <styled.span
          onClick={[Function]}
        >
          In album: 
        </styled.span>
      </styled.div>
    </styled.div>
  </styled.div>
  <styled.div>
    <styled.div>
      <styled.div>
        <t
          ago={false}
          calendar={false}
          decimal={false}
          element="time"
          filter={[Function]}
          fromNow={true}
          interval={60000}
          onChange={[Function]}
          toNow={false}
          unit={null}
          unix={false}
          utc={false}
        />
      </styled.div>
      <styled.span
        onClick={[Function]}
      >
        # 
      </styled.span>
    </styled.div>
  </styled.div>
  <styled.div
    onClick={[Function]}
  >
    <styled.img
      src="https://s3-us-west-1.amazonaws.com/streamboard98/albumCovers/undefined"
    />
  </styled.div>
</styled.div>
`;
